client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -194,16 +204,38 @@ const MetadataEditor = createClass({
 	},
 
 	renderAuthors : function(){
+		const authors = this.props.metadata.authors;
 		let text = 'None.';
-		if(this.props.metadata.authors && this.props.metadata.authors.length){
-			text = this.props.metadata.authors.join(', ');
+		if(authors && authors.length){
+			text = authors.join(', ');
 		}
-		return <div className='field authors'>
-			<label>authors</label>
-			<div className='value'>
-				{text}
+		if(!this.state.isOwner || authors.length < 2) return (
+			<div className='field authors'>
+				<label>authors</label>
+				<div className='value'>
+					{text}
+				</div>
 			</div>
-		</div>;
+		);
+		return (
+			<div className='field authors'>
+				<label>authors</label>
+				<ul className='list'>
+					{authors.map((author, i)=>(
+						<li className='tag' key={i}>{author}
```

Comment:
Instead of conditionally rendering the 'remove' button, you might consider conditional rendering the entire tag:  either an "owner tag" or a "author tag".  Then, only author tags would have the remove button.

This would allow you to give the "owner tag" a `title: "Owner"` attribute, which would make it more clear why that tag is special and why it is has different styling.  It would also be more straightforward for the styling itself, you could give each a `.owner` or `.author` tag and apply CSS based on that rather than using `:nth-child(1)`.  

Also, each of these author tags could be a link to that users profile page.